docs/source/teaching-checklist.md

```diff
@@ -0,0 +1,95 @@
+# Checklist for a JupyterHub teaching deployment
+
+Documentation for teaching deployment: https://jupyterhub-deploy-teaching.readthedocs.io
+
+Documentation for JupyterHub: https://jupyterhub.readthedocs.io
+
+## Notes
+
+- Does **not** use Docker.
+- [NGINX](https://www.nginx.com) as a frontend proxy, for serving static
+  assets, and a termination point for SSL/TLS.
+- Single Ubuntu server
+- [Ansible](https://www.ansible.com/resources) for configuration.
+
+## 1. Prepare the server
+
+- [ ] **Server:** running latest Ubuntu version
+- [ ] **SSH:** enable password-less SSH for `ubuntu` user
+- [ ] **Local disks:** partition and format
+- [ ] **DNS (domain name):** valid entry for server
+- [ ] **SSL certificate/key:** obtain via let's encrypt or trusted source
```

Comment:
The letsencrypt step is done by ansible, so it should be clear that you don't need to have SSL setup before you start if using letsencrypt. I see this is noted in step 3, but I don't want people to get held up here.